packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js

```diff
@@ -58,6 +58,9 @@ export const STORAGE_GATEWAY = {
   US: 'gateways.storage.REGION.cloud.ovh.us',
 };
 
+export const STORAGE_PRICES_LINK =
```

Comment:
only FR link? what about other supported languages? can you add support to fetch the link based on user subsidiary? if a link for the given subsidiary is not available, default it back to FR.

Reply:
Good point, I'll ask